src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -105,6 +128,19 @@ public class Utils {
   public static final String GET_EXCEPTION_MISSING_MESSAGE = "missing exception message";
   public static final String GET_EXCEPTION_MISSING_CAUSE = "missing exception cause";
 
+  // OAuth
+  public static final String TOKEN_REQUEST_ENDPOINT = "/oauth/token-request";
+  public static final String OAUTH_CONTENT_TYPE_HEADER = "application/x-www-form-urlencoded";
+  public static final String BASIC_AUTH_HEADER_PREFIX = "Basic ";
+  public static final String GRANT_TYPE_PARAM = "grant_type";
+  public static final String REFRESH_TOKEN = "refresh_token";
+  public static final String ACCESS_TOKEN = "access_token";
+  public static final String SNOWFLAKE_JWT = "snowflake_jwt";
+  public static final String OAUTH = "oauth";
+  public static final String REDIRECT_URI = "redirect_uri";
+  public static final String DEFAULT_REDIRECT_URI = "https://localhost.com/oauth";
```

Comment:
i wish we could use a library. can we? are these standard terms used in all oauth authentications?

Reply:
Some of these are snowflake specific, moved to `OAuthConstants.java`